src/emcpy/plots/tools.py

```diff
@@ -0,0 +1,94 @@
+class ColorbarPreset:
+    """
+    Class that has pre-determined vmin, vmax,
+    and cmap values depending on the variable
+    and evaluation type.
+    """
+    def __init__(self, variable, eval_type):
+        """
+        Class constructor.
+
+        Args:
+            variable : (str) variable of interest
+            eval_type : (str) evaluation type
+        """
+
+        cbar_d = {
+            'brightness temperature': {
+                'diff': {
+                    'vmax': -5,
```

Comment:
vmax is -5?

Reply:
mixed up the vmin/vmax numbers for differences. Will fix.